README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
Well, I probably took it out, because the more recent releases don't have this package anymore - it is now part of the ghc package and thus it is "automagically" there. However, because you bypass the system, it hits you right into the face ...

BTW: Take care about what you add to the debian/control file. If it is not sufficient or invalid, this will break launchpad builds - and dealing with launchpad errors is horror at its best/costs a lot of time! So make sure, that at least for the main target (current LTS release) all specified packages actually exist and are sufficient to build the package! launchpad is an extremely picky ...